network-store-model/src/main/java/com/powsybl/network/store/model/TemporaryCurrentLimitAttributes.java

```diff
@@ -22,14 +23,32 @@
 @Schema(description = "Temporary current limit attributes")
 public class TemporaryCurrentLimitAttributes {
```

Comment:
Now that there are multiple type of TemporaryLimits, maybe we should rename this class `TemporaryCurrentLimitAttributes` to `TemporaryLimitAttributes` because it's not always a `Current` one...

Reply:
Ok